### DISCUSSION TITLE
Chat space

### DISCUSSION THREAD

**Discussion by DuzzuAcid** [jellyfin/jellyfin#8059] [General]
Hi everyone, could it be possible to have a public chat space, maybe to open clicking on a button positioned somewhere up-right so we can talk with our users and they can talk within themselves, it would be nice to have achievements based on genres, such as "horror lover" if someone viewed 50 movies tagged as horror and options to share what someone is viewing or has just finished to view and let others know so that we can talk about it.

**Comment by cvium** [comment]
I believe I can safely say that that will forever be out of scope for Jellyfin. It's a media server, not a community platform.

I think you can get a long way with a Discord server and the webhook plugin though, but it's not something we'd want as part of Jellyfin.

**Comment by oddstr13** [comment]
The discussion about and voting on a potential chat feature belongs in this thread;
https://features.jellyfin.org/posts/740/add-chat-to-syncplay
